xml_parent_id.py

```diff
@@ -0,0 +1,71 @@
+import xml.etree.ElementTree as ET
```

Comment:
So this is a script that transformed all the xmls?  Can we use it for fw-ports as well? (automatically)  I suppose this needs to be cleaned out...

Reply:
Yes, I just put it there to be sure to not lose it.
I will put it in my gists later if it is needed.

It could be used to handle the fw-ports if needed